src/core/router/net_db/impl.h

```diff
@@ -80,10 +80,27 @@ enum struct NetDbInterval : const std::uint16_t {
   DelayedExploratory = 90,
 };
 
+/// @enum NetDbTime
+/// @brief Constants defining timestamp
+///  variables for various NetDb operations
+enum struct NetDbTime : const std::uint64_t {
+  /// @var RouterExpiration
+  /// @brief in milliseconds
+  RouterExpiration = 3600 * 1000LL,
+  /// @var RouterStartupLimit
+  /// @brief in milliseconds, defines
+  ///  a grace period when a router
+  ///  has just started to not set
+  ///  expired routers as unreachable,
+  ///  so tunnels will be built quickly
+  RouterStartupPeriod = 600 * 1000LL,
```

Comment:
`LL` is not needed here but if you revert the 387-389 work I noted above, it *will* be needed within *that* context (unless you refactor another way).